modules/administration/pages/iss_v2.adoc

```diff
@@ -72,4 +72,5 @@ Properties file location can be overridden with parameter ``--serverConfig``.
 == Known Limitations
 
 * Source and target servers need to be on the same version.
-* Export and import organization should have the same name.
+* Export and import organization should have exactly the same name.
+  The organization name is case sensitive.
```

Comment:
```suggestion
  * The organization name is case sensitive.
```

Reply:
I'd make the case sensitive requirement more pronounced by turning into a bullet-point (no need to the admonition though).

Reply:
I think this is not needed.